sos/report/plugins/foreman.py

```diff
@@ -244,8 +244,17 @@ def setup(self):
             self.add_cmd_output(_cmd, suggest_filename=table, timeout=600,
                                 sizelimit=100, env=self.env)
 
+        # dynflow* tables on dynflow >=1.6.3 are encoded and hence in that
+        # case, psql-msgpack-decode wrapper tool from dynflow-utils (any
+        # version) must be used instead of plain psql command
+        dynutils = 'dynflow-utils' in \
+            self.policy.package_manager.get_pkg_list().keys()
```

Comment:
Is `self.is_installed('dynflow-utils')` not sufficient here?

Reply:
Ah right. It was a relic of previous testing "dynutils of version .." which is not required.